fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1153,6 +1153,24 @@
     @ConfField
     public static String auth_token = "";
 
+    /**
+     * If set to true and the jar that use to authentication is loaded in fe, kerberos authentication is supported.
+     */
+    @ConfField(mutable = true)
+    public static boolean authentication_kerberos_enabled = false;
+
+    /**
+     * Service principal name. like "starrocks-fe/hostname@STARROCKS.COM"
+     */
+    @ConfField(mutable = true)
+    public static String authentication_kerberos_service_principal = "";
```

Comment:
Why is this config needed? Should add more detail explanation.

Reply:
done